tests/test_syntax/test_sequential2.py

```diff
@@ -170,25 +170,6 @@ def __call__(self) -> m.Bits[8]:
                              f"build/TestSequential2DefaultClock.v",
                              f"gold/TestSequential2DefaultClock.v")
 
-    @m.sequential2()
```

Comment:
why is this removed?

Reply:
This test relied on implicitly wiring the first clock when there were multiple possible drivers.  This is the behavior that this change set is seeking to avoid, so we can remove this test.  I think when this was added we were considering hte possibility of just automatically wiring up the first driver as default, but I think the changes added here where we don't wire anything up is preferred.  I don't think anyone is using this feature.